components/eventing-controller/pkg/backend/nats/jetstream/jetstream.go

```diff
@@ -29,12 +29,15 @@ import (
 var _ Backend = &JetStream{}
 
 const (
-	jsHandlerName          = "jetstream-handler"
-	idleHeartBeatDuration  = 1 * time.Minute
-	jsConsumerMaxRedeliver = 100
-	jsConsumerAcKWait      = 30 * time.Second
-	jsMaxStreamNameLength  = 32
-	separator              = "/"
+	jsHandlerName                   = "jetstream-handler"
+	idleHeartBeatDuration           = 1 * time.Minute
+	jsConsumerMaxRedeliver          = 100
+	jsConsumerAcKWait               = 30 * time.Second
+	jsMaxStreamNameLength           = 32
+	separator                       = "/"
+	MissingNATSSubscription         = "failed to create NATS JetStream subscription"
```

Comment:
do you need to export this variable?
```suggestion
	missingNATSSubscriptionMsg         = "failed to create NATS JetStream subscription"
```